Commit #500 breaks code using ExternalProject_Add(): "... first defined here"

**Describe the bug**
Since commit #500 compilation of my code that uses HighFive fails on linking, with multiple definitions of `HighFive::Group::getEstimatedLinkInfo()`, `HighFive::File::getMetadataBlockSize()` and  `HighFive::File::getVersionBounds()`. Seems like the inline specifier is missing?

**To Reproduce**
I fetch HighFive via
```include(ExternalProject)
ExternalProject_Add(HighFive
        GIT_REPOSITORY https://github.com/BlueBrain/HighFive
	#GIT_TAG f9eb793d31ccd706a5ed2441d096a5c8ca1581e5
        LOG_DOWNLOAD ON
        CMAKE_ARGS -DHIGHFIVE_USE_BOOST=OFF
        CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${HF_PREFIX}
        CMAKE_ARGS -DHIGHFIVE_EXAMPLES=OFF
        CMAKE_ARGS -DHIGHFIVE_PARALLEL_HDF5=ON
        )
```

**Expected behavior**
The code should compile and link as usual. However, it fails on linking. If I uncomment #GIT_TAG f9eb793d31ccd706a5ed2441d096a5c8ca1581e5 (commit before #500) everything works as expected.

**Stacktrace**
```
(.text+0x0): multiple definition of `HighFive::Group::getEstimatedLinkInfo() const'; CMakeFiles/Iltis.dir/main.cpp.o (symbol from plugin):(.text+0x0): first defined here
(.text+0x0): multiple definition of `HighFive::File::getMetadataBlockSize() const'; CMakeFiles/Iltis.dir/main.cpp.o (symbol from plugin):(.text+0x0): first defined here
(.text+0x0): multiple definition of `HighFive::File::getVersionBounds() const'; CMakeFiles/Iltis.dir/main.cpp.o (symbol from plugin):(.text+0x0): first defined here
```

**Desktop (please complete the following information):**
 - Debian Bullseye
 - Version #500

--- a/include/highfive/H5Group.hpp
+++ b/include/highfive/H5Group.hpp
@@ -43,7 +43,7 @@ class Group : public Object,
 
 #include <H5Gpublic.h>
 
-std::pair<unsigned int, unsigned int> Group::getEstimatedLinkInfo() const {
+inline std::pair<unsigned int, unsigned int> Group::getEstimatedLinkInfo() const {
     unsigned int est_num_entries;
     unsigned int est_name_len;
 

--- a/include/highfive/bits/H5File_misc.hpp
+++ b/include/highfive/bits/H5File_misc.hpp
@@ -84,7 +84,7 @@ inline const std::string& File::getName() const noexcept {
     return _filename;
 }
 
-hsize_t File::getMetadataBlockSize() const {
+inline hsize_t File::getMetadataBlockSize() const {
     hsize_t size;
     auto fid_fapl = H5Fget_access_plist(getId());
     if (H5Pget_meta_block_size(fid_fapl, &size) < 0) {
@@ -94,7 +94,7 @@ hsize_t File::getMetadataBlockSize() const {
     return size;
 }
 
-std::pair<H5F_libver_t, H5F_libver_t> File::getVersionBounds() const {
+inline std::pair<H5F_libver_t, H5F_libver_t> File::getVersionBounds() const {
     H5F_libver_t low;
     H5F_libver_t high;
     auto fid_fapl = H5Fget_access_plist(getId());

